### ISSUE TITLE
[Bug] [1.18.2] Blocks frequently turn invisible when building into empty subchunks

### ISSUE THREAD

**Issue by LordNadiki** [sp614x/optifine#6751]
## Description of Issue
Often when building upwards, sideways, or growing tall trees, the blocks will not visually update, resulting in invisible ghost blocks. They have hitboxes and can be interacted with, but it takes around 3-15 seconds for them to appear. It always happens at subchunk borders, and seemingly happens more often higher up (Y:144+).

## Steps to Reproduce
1. Open a new or existing world.
2. Find a clear, flat area.
3. Pillar up or build sideways into an empty subchunk with some building blocks, or grow a tall tree just below an empty subchunk.

## OptiFine Version
1.18.2 OptiFine HD U H7 (also occurs in 1.18.2 OptiFine HD U H6)

## Installation Method
Installed using the standalone OptiFine installer.

## Other Installed Mods
No other mods are installed.

## F3 Debug Screenshots
![2022-04-12_01 32 37](https://user-images.githubusercontent.com/24702037/162929249-0e00cb7d-f679-4067-b166-9f542ba8dcd2.png)
![2022-04-12_01 14 33](https://user-images.githubusercontent.com/24702037/162929258-0fd22b9b-5fd6-4f19-841f-e5b380824de0.png)

## Prior Testing
I am unable to replicate this in Vanilla.

## Additional Information
I tried every different setting of Render Regions, Fast Render, Chunk Builder, and Chunk Updates, and none of them affected the frequency or duration of the invisible blocks.

**Comment by Tohricat** [comment]
To add - I've noticed this seems to occur most frequently when building a single block column, pillar or row in any direction into an empty subchunk. Slowly building up a 5x1 wall for example, into an empty subchunk, is unlikely to trigger the bug.

**Comment by Kurisutian123** [comment]
Exactly, I have the same problem.

The problem only occurs (randomly) when I have "Chunk Builder: Threaded" activated.
Problem is fixed for me when I set it to "Chunk Builder: Semi-Blocking".

Could you please confirm this solves it for you too?
This way the developers will have an easier time fixing this obvious bug.

My CPU is the Ryzen 2400G
My GPU 1660 TI 6GB

**Comment by LordNadiki** [comment]
Just tested, it still happens with Semi Blocking, but is somewhat less frequent than Threaded. Fully Blocking is about the same as Semi Blocking.

**Comment by Pause-Kawa** [comment]
Same issue for me, with Optifine 1.19.2-HD-U-H9.
Still no fix or way to prevent that?

![2022-12-09_00 13 14](https://user-images.githubusercontent.com/60948025/206596885-4d7e42ee-9cae-455b-918f-a4f2b9dbf3bd.png)

**Comment by scienide00** [comment]
issue happens on OptiFine_1.19.2_HD_U_H9 as well

**Comment by darthvader45** [comment]
Same.  Rather annoying.

**Comment by trathborne** [comment]
I've seen it in 1.19.4 HD U I3 pre[1-5] too.

When tunneling into a new area, it isn't rendered even if I dig out some blocks, until I step into the new area.

**Comment by darthvader45** [comment]
Yeah, this is getting out of hand.  Sooner or later we need to not have such bugs happen so often.

**Comment by WindowsShadows** [comment]
Happens in 1.19.3 HD U I2 pre2

**Comment by IpanXBread** [comment]
Still happens in 1.20.1

**Comment by BurgerXXL420** [comment]
Got the same problem. Still playing on OptiFine_1.19.2_HD_U_H9
